<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administracao_estoque</class>
 <widget class="QMainWindow" name="administracao_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administração de estoque</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="lista_estoques">
     <attribute name="title">
      <string>Estoques</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_estoque">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>50</y>
        <width>400</width>
        <height>181</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QWidget" name="layoutWidget_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>381</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_novo_estoque_2">
         <property name="text">
          <string>Novo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_gravar_estoque_2">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remover_estoque_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Remover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_voltar_estoque_2">
         <property name="text">
          <string>Voltar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>12</y>
        <width>397</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pesquisar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pesquisar_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="lista_produtos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Produtos</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_produtos">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>50</y>
        <width>400</width>
        <height>181</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>12</y>
        <width>397</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pesquisar_descricao">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pesquisar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>381</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_remover">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remover</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="administracao">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Administração</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_estoque">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>38</width>
        <height>24</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_ativo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ativo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>251</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Finalidade *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_finalidade"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>381</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_novo_estoque">
         <property name="text">
          <string>Novo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_gravar_estoque">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remover_estoque">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Remover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_voltar_estoque">
         <property name="text">
          <string>Voltar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>110</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_nome"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_pesquisar</tabstop>
  <tabstop>pushButton_pesquisar_2</tabstop>
  <tabstop>tableWidget_estoque</tabstop>
  <tabstop>pushButton_novo_estoque_2</tabstop>
  <tabstop>pushButton_gravar_estoque_2</tabstop>
  <tabstop>pushButton_remover_estoque_2</tabstop>
  <tabstop>pushButton_voltar_estoque_2</tabstop>
  <tabstop>lineEdit_pesquisar_descricao</tabstop>
  <tabstop>pushButton_pesquisar</tabstop>
  <tabstop>tableWidget_produtos</tabstop>
  <tabstop>pushButton_remover</tabstop>
  <tabstop>lineEdit_id_estoque</tabstop>
  <tabstop>radioButton_ativo</tabstop>
  <tabstop>lineEdit_nome</tabstop>
  <tabstop>lineEdit_finalidade</tabstop>
  <tabstop>pushButton_novo_estoque</tabstop>
  <tabstop>pushButton_gravar_estoque</tabstop>
  <tabstop>pushButton_remover_estoque</tabstop>
  <tabstop>pushButton_voltar_estoque</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
